recipes/IWSLT22_lowresource/README.md

```diff
@@ -0,0 +1,56 @@
+# IWSLT 2022 Low-resource Task: Tamasheq-French end-to-end Speech Translation
+
+
+## Description
+
+This is the recipe for the best system from the IWSLT 2022 low-resource task, as described in the original paper.
+The speech translation model comprises a wav2vec 2.0 encoder and a Transformer decoder. It is trained end-to-end without any auxiliary loss. The recipe allows for removing the last layers of the Transformer Encoder inside the wav2vec 2.0 in order to reduce the number of training parameters.
+
+## Data Downloading
+
+For downloading the dataset used for this experiment, please run the following command.
+
+```
+git clone https://github.com/mzboito/IWSLT2022_Tamasheq_data.git
+```
+
+## Training
+
+For training the model, please update the variables at hparams/train_w2v2_st.yaml.
+
+Note that in order to drop the last layers of the wav2vec 2.0 module, it is necessary to update the parameter "keep_n_layers".
+For instance: Using ``keep_n_layers: 10'' means that only the first 10 layers inside the wav2vec 2.0 Transformer encoder will be used for training. The remaining layers are removed.
+
+For launching training:
+```
+python train.py hparams/w2v2_st.yaml
```

Comment:
@mzboito the yaml file name changed (sorry I spot it only now) -> hparams/train_w2v2_st.yaml

Reply:
Nice catch! I fixed it in my last commit. Thanks! :)